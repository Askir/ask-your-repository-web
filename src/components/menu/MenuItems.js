// @flow
import React from 'react';
import './Menu.scss';

type Props = {
  close: Function,
};

function MenuItems(props: Props) {
  return (
    <div className="menu">
      <ul>        
        <li onClick={props.close}><a href="./">Add New Artifact</a></li>
        <li onClick={props.close}><a href="./images">View All</a></li>
      </ul>
    </div>
  );
}

export default MenuItems;
