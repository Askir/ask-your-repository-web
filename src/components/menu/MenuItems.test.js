// @flow
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MenuItems from './MenuItems';

describe('<MenuItems />', () => {
  let wrapper;

  beforeEach(() => {
    const close = jest.fn();
    wrapper = shallow(<MenuItems close={close} />);
  });

  it('renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
