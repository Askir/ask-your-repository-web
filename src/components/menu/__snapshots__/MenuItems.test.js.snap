// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItems /> renders correctly 1`] = `
<div
  className="menu"
>
  <ul>
    <li
      id="btnUpload"
      onClick={[MockFunction]}
    >
      <a
        href="./"
      >
        Add New Artifact
      </a>
    </li>
    <li
      id="btnViewAll"
      onClick={[MockFunction]}
    >
      <a
        href="./images"
      >
        View All
      </a>
    </li>
  </ul>
</div>
`;
